library/NotificationCenter/Gateway/Email.php

```diff
@@ -48,11 +48,17 @@ public function createDraft(Message $objMessage, array $arrTokens, $strLanguage
         return new EmailMessageDraft($objMessage, $objLanguage, $arrTokens);
     }
 
-    /**
-     * @param EmailMessageDraft $objDraft
-     */
-    public function sendDraft(EmailMessageDraft $objDraft)
+    private function instantiateEmail()
     {
+        if (version_compare(VERSION, '4.10', '>=')) {
```

Comment:
In theory it's still possible to run 4.10 with swiftmailer, isn't it? That's why Contao's `Email` works with both.

Reply:
Then we have to check the type of `mailer` I guess.